### DISCUSSION TITLE
Adding extensions (Dockerfile)

### DISCUSSION THREAD

**Discussion by eatenpancreas** [zed-industries/zed#8077] [Feature Requests]
According to [this comment](https://github.com/zed-industries/zed/issues/5311#issuecomment-1952372105) and [this pull request](https://github.com/zed-industries/extensions/pull/60) Dockerfile should now be supported as an extension.

How do i enable this? Or do i need to wait for next release? There's not much documentation on how extensions work. Thanks in advance! :)

**Comment by SomeoneToIgnore** [comment]
It is actually not an extension even, but a part that ships with Zed (for now: due to the fact that extensions do not have any LSP support yet). 

That is supposed to be released as part of Zed 0.124, preview version of which will be released tomorrow.
It will be promoted to stable next week's Wednesday.
